aiida_firecrest/transport.py

```diff
@@ -135,16 +282,29 @@ def __init__(
         url: str,
         token_uri: str,
         client_id: str,
-        client_secret: str | Path,
+        client_secret: str,  # | Path,
+        # unfortunately we cannot store client_secret as a Path, because it is not JSON serializable
         client_machine: str,
-        small_file_size_mb: float = 5.0,
-        file_transfer_poll_interval: float = 0.1,
+        small_file_size_mb: float,
+        temp_directory: str,
+        # configured: bool = True,
```

Comment:
why commented out?

Reply:
thanks for noticing, we don't need this anymore actually. to be removed
```suggestion
```